### PR TITLE
Set region awareness envvar to true in Cassandra when region is specified

### PR DESCRIPTION
**Pull request by mrbrowning**

### CODE REVIEW

File: frameworks/cassandra/universe/marathon.json.mustache [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "VIRTUAL_NETWORK_PLUGIN_LABELS": "{{service.virtual_network_plugin_labels}}",
     {{/service.virtual_network_enabled}}
     {{#service.region}}
+    "ALLOW_REGION_AWARENESS": "true",
```
Comment (gabrielhartmann) [inline frameworks/cassandra/universe/marathon.json.mustache]: Why have this envvar at all if it's hard-coded to true?  Couldn't it just be added to Cassandra's Scheduler config programmatically?

**Comment by mrbrowning** [comment]
@gabrielhartmann just to save a step, since the one implies the other, and otherwise the user would have to specify some `config.allow_region_awareness` in addition to specifying a remote region if they wanted to run in one.

File: frameworks/cassandra/universe/marathon.json.mustache (hunk 1 above)
Comment (mrbrowning) [reply frameworks/cassandra/universe/marathon.json.mustache]: Oh, I see what you mean. I suppose it could.

Comment (mrbrowning) [reply frameworks/cassandra/universe/marathon.json.mustache]: Well, `SchedulerConfig` would have to be modified to support overrides.

**Pull request merged**